static/sass/_snapcraft_p-release-details-row.scss

```diff
@@ -11,7 +11,7 @@
       grid-template-columns: auto;
     }
 
-    &:nth-child(odd) {
+    &:nth-child(even) {
```

Comment:
I recall some "no zebra stripes" design rule for tables, don't remember where...

Reply:
MAAS did a spike exploring zebra strips but not sure on the outcome of it... https://github.com/canonical-web-and-design/MAAS-design/issues/524#issuecomment-415693413

Reply:
Is this a Mantra? :eyes: @anthonydillon

Reply:
The drawback, of zebra stripes for table rows, is that the table needs at least four rows for people to have a chance of realising that they _are_ zebra stripes — rather than just one item being selected and the others not. Even if some tables ~always had at least four items, this one definitely won’t.

Reply:
Would each row be OK with the background simply removed? Or would we need to add horizontal lines?